copier.yml

```diff
@@ -175,6 +175,12 @@ github_enable_codecov:
   help: GitHub action runs codecov/codecov-action?
   when: *ci_is_github
 
+github_v4_codecov:
```

Comment:
Should this be a question? I think it should be the only option.

Reply:
I can, this is simpler. @sbidoul

Reply:
Since we already have github_enable_codecov, I don't think a new question is necessary.

Reply:
Ok, that leads to this change :smile: :

![image](https://github.com/OCA/oca-addons-repo-template/assets/19529533/0d0512e6-f464-4da3-b0a5-265b035d9c7d)

Reply:
Yeah I had not checked, so I did not know we had a question for codecov already.